recordHooks/employees/employeeValidations.ts

```diff
@@ -42,13 +43,6 @@ export function employeeValidations(record) {
     // Handle or rethrow the error as needed
   }
 
-  try {
-    formatRecordDates(record, 'employees-sheet');
-  } catch (error) {
-    console.log('Error occurred during date formatting:', error);
-    // Handle or rethrow the error as needed
-  }
-
```

Comment:
Do we not need to validate these dates anymore?

Reply:
@colinfrederickson